site/Calendar.php

```diff
@@ -0,0 +1,29 @@
+<html>
+	<head>
+	<link rel="stylesheet" type="text/css" href="Style.css">
+	<link href="https://fonts.googleapis.com/css?family=Raleway"
+	rel="stylesheet">
+		<title>
+			St. Louis Theatre Calendar
+		</title>
+	</head>
+	<body>
+		<div class="Header">
+			<?php include 'include/header.php';
```

Comment:
I'd suggest a few changes here:

1- Always put includes at the top of the page unless there's a good reason not to. That way you can easily tell what all the dependencies are in one places instead of needing to hunt around in the code for the includes

2- have your init.php code include header.php, and then you only need to include init.php at the top. As you go forward, there will be more and more libraries you need to include and it becomes a mess if you're doing each one manually. 99% of pages should only have one include, and that's init.php.

3- I'd suggest using `include_once` instead of include. This way if you accidentally include something more than once, PHP is smart enough to ignore the second time.